src/api/getDairaByBaladyiaName.js

```diff
@@ -0,0 +1,36 @@
+const projectWilaya = require("../utils/projections/wilayaProjection");
+
+const _hasName = baladyiaName =>
```

Comment:
Please consider exporting this to `utils`, and replace it in places we use the same logic